webapp/src/components/Settings/DiscardButton.tsx

```diff
@@ -0,0 +1,19 @@
+import { Replay } from "@mui/icons-material";
+import { InputAdornment, Tooltip, IconButton } from "@mui/material";
+
+export const DiscardButton: React.FC<{
+  title: string;
+  disabled: boolean;
+  onClick: () => void;
+}> = ({ title, ...props }) => (
+  <InputAdornment position="start">
+    <Tooltip
+      placement="bottom-start"
+      title={props.disabled ? "" : `Discard change, reset to ${title}`}
```

Comment:
I'm wondering if I should add more, like
```suggestion
      title={props.disabled ? "" : `The blue arrow indicates a change. Click to reset to ${title}`}
```

Reply:
Personally I like the tooltip as is